fe1-web/model/objects/wallet/Wallet.ts

```diff
@@ -1,10 +1,11 @@
 import { useSelector } from 'react-redux';
-import { makeEventByTypeSelector } from 'store';
+import { RollCall } from 'features/rollCall/objects';
```

Comment:
same here, it'll require a bit of re-engineering, but we may be able to get away with not depending on the roll call. 
For instance, we could pass the roll calls as a parameter to the function.